packages/backend-modules/publikator/script/migrateAudioSources.ts

```diff
@@ -102,7 +104,7 @@ const handleBatch = async (rows: any[], count: number, pgdb: any) => {
       return
     }
 
```

Comment:
This script probably woudln't work correctly like this, you would also need to check for audioSourceByteSize in the checks above. But afaik this script was used to add a synthetic voice to all past articles, so probably it isn't that relevant